app-vim/jedi/jedi-0.10.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+
+inherit vim-plugin python-r1
```

Comment:
Just by looking at the ebuild and installed files... I don't think *python-r1.eclass* is the correct one here. I'd use *python-any-r1.eclass* instead.